sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClientBuilder.java

```diff
@@ -104,6 +104,7 @@ public final class TableClientBuilder implements
     private TableServiceVersion version;
     private RetryPolicy retryPolicy;
     private RetryOptions retryOptions;
+    private boolean enableTenantDiscovery = false;
```

Comment:
By default, `boolean` vars are false. So, you don't need to explicitly assign `false` here.